app/services/decisions/income_decision_tree.rb

```diff
@@ -105,16 +125,44 @@ def ended_employment_within_three_months?
       form_object.ended_employment_within_three_months&.yes?
     end
 
+    def appeal_no_changes?
+      crime_app.case.case_type == CaseType::APPEAL_TO_CROWN_COURT.to_s
+    end
+
+    def summary_only?
+      crime_app.case.case_type == CaseType::SUMMARY_ONLY.to_s
+    end
+
     def income_below_threshold?
-      form_object.income_above_threshold.no?
+      crime_app.income.income_above_threshold == 'no'
     end
 
     def no_frozen_assets?
-      form_object.has_frozen_income_or_assets.no?
+      crime_app.income.has_frozen_income_or_assets == 'no'
```

Comment:
Again, what is the purpose of this change? Do we have a type casting issue?

Reply:
Because originally these were helper methods just used by the forms that collected those values, so could be accessed from the form_object. I now need them on later forms, so thee data is on the crime_application.income, not on the form_object